src/style.css

```diff
@@ -1 +1,79 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf'), url('fonts/Avenir-Heavy.ttf');
+  font-weight: 300;
+}
+
+body {
+  margin: 0;
+}
+
+.search__bar {
+  margin: 20px 8px 0;
+}
+
+.search__bar--input {
+  width: 100%;
+  font-family: Avenir, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. You've provided a fallback of 'sans-serif', which is good, but ensure that the font-family 'AvenirHeavy' used in the focus state also has a similar fallback.